selenium_tests/lti_emailer/page_objects/canvas_masquerade_page_object.py

```diff
@@ -0,0 +1,35 @@
+from selenium.webdriver.common.by import By
+from selenium_tests.lti_emailer.page_objects.emailer_base_page_object import EmailerBasePageObject
+from urlparse import urljoin
+
+
+class CanvasMasqueradePageLocators(object):
+    CONFIRM_BUTTON = (By.CSS_SELECTOR, "a.btn.masquerade_button")
+    MASQUERADE_URL = 'users/{}/masquerade'
+
+
+class CanvasMasqueradePageObject(EmailerBasePageObject):
```

Comment:
Masquerade isn't specific to the lti_emailer, can we put this in base page object?